index.html

```diff
@@ -1762,79 +1769,103 @@ <h1>WoT Building Blocks</h1>
 			The WoT Building Blocks support each of the architectural aspects of
 			a Thing discussed in <a href="#sec-arch-aspects"></a> and diagrammed
 			in <a href="#arch-aspects"></a>. The building blocks themselves are
-			shown in <a href="#arch-thing-with-scripting"></a>.
+			shown in the context of a single Thing in <a href="#arch-thing-with-scripting"></a>.
 		<figure id="arch-thing-with-scripting">
-			<img src="images/wot-thing-with-scripting.png" />
+			<img src="images/wot-thing-runtime-scripting.png" />
 			<figcaption>Thing with WoT Building Blocks, including
-				the WoT Scripting API</figcaption>
+				the WoT Runtime and the WoT Scripting API</figcaption>
 		</figure>
 		</p>
 		<p>The WoT Thing Description contains metadata describing the
-			affordances of the Thing, the mapping of these affordances onto
-			concrete network protocols, and the public security metadata needed
-			by authorized users to access the Thing. For specifying behavior, an
-			optional WoT Scripting API is provided which supports the definition
-			and consumption of WoT Thing Descriptions. This is useful when the
-			application behavior is defined in a scripting language.</p>
+			interaction affordances of the Thing, the mapping of these affordances onto
+			concrete network protocols (the communication metadata), 
+                        the public security data needed
+			by authorized users to access the Thing,
+                        and other general metadata such as IDs, names, and descriptions.
+                        A Binding Template provides guidance for providing the correct communications metadata
+                        in a Thing Description to support a specific target protocol.
+                        For specifying behavior, an
+			optional WoT Scripting API building block is provided which supports the definition
+			and consumption of WoT Thing Descriptions. This is useful to define
+			application behavior using and easy-to-use scripting language.</p>
 		<p>
-			It is also possible, as shown in <a
-				href="#arch-thing-without-scripting"></a>, to use the WoT Thing
-			Description to described a device or service that does not use the
-			WoT Scripting API. There are two cases where this may be important:
-			pre-existing "brownfield" devices for which we are providing a Thing
-			Description in order to allow them to interoperate with other
-			elements of a WoT system; and devices which need to use a programming
-			language not supported by the WoT Scripting API.
+                <p>It is however possible to implement a WoT Thing <em>without</em> using the WoT Runtime
+                and WoT Scripting API as shown in <a href="#arch-thing-without-scripting"></a>.
```

Comment:
Somehow we need to make clear that the WoT Runtime is larger concept than the Scripting Runtime used in the implementation of the Scripting API.

Reply:
Yes, I think your comments were a little behind my edits.